src/content/ulysses/index.html

```diff
@@ -0,0 +1,30 @@
+---
+title: Ulysses
+color: purple
+layout: theme.html
+icon: candle
+icon_width: 198
+icon_height: 184
+---
+
+<img class="preview" src="{{ baseurl }}/assets/img/screenshots/ulysses.png" alt="Theme Preview" width="3378" height="2116">
+
+<div class="instructions">
+	<h3>Ulysses</h3>
+
+	<h4>Ulysses Style Exchange</h4>
+
+	<p>You can get Dracula from the <a href="http://styles.ulyssesapp.com/bundle/Dracula/56319778b634a4410207d126">Ulysses Style Exchange</a>.</p>
+
+	<h4>Install manually</h4>
+
+	<p>Clone or <a href="https://github.com/dracula/ulysses.git/archive/master.zip">download</a> the repo, then open the Dracula.ultheme file.</p>
```

Comment:
Please see some of the other themes on how they handle this. We like to put the exact instructions on how to clone the repo.